Update documentation for IS 7.0.0 onwards (#31)
wso2-extensions/identity-outbound-auth-linkedIn: docs/is7/README.md

# Configuring LinkedIn Authenticator In Identity Server (7.0.0 Onwards)

This guide provides instructions on how to configure the LinkedIn authenticator and add LinkedIn login to your applications using WSO2 Identity Server, enabling users to log in with their LinkedIn accounts.

Follow the steps given below to set this up.

## Configure the LinkedIn Authenticator

1. Download the .jar file from the [WSO2 Store](https://store.wso2.com/connector/identity-outbound-auth-linkedIn)
    - **OAuth 2.0** : `org.wso2.carbon.extension.identity.authenticator.linkedin.connector-x.x.x` 
    - **OpenID Connect (V2)** : `org.wso2.carbon.extension.identity.authenticator.linkedin.oidc.connector-x.x.x`
2. Place the authenticator.jar file into the 
`<IS_HOME>/repository/components/dropins` directory. 

3. Start or restart the WSO2 Identity Server. 

> > If you want to upgrade the LinkedIn (.jar) in your existing Identity Server pack, please refer [upgrade instructions](https://docs.wso2.com/display/ISCONNECTORS/Upgrading+an+Authenticator).

Reviewer: Is this link working?

Author: fixed

## Register WSO2 Identity Server on LinkedIn

1. Navigate to LinkedIn [Create an app page](https://www.linkedin.com/developer/apps/new).

2. Enter the required details.
    * App Name and Company details. The company should have a LinkedIn page.
    * Upload an image that you wish to use as the company logo.
    * Select the checkbox to agree to the LinkedIn terms and conditions.

> 3. Click on **Create App**. You will be redirect to a page with **Settings** for the created application. 

Reviewer: ```suggestion
3. Click on **Create App**. You will be redirected to the  **Settings** page of the created application. 
```

Author: fixed
